modules/300-prometheus/openapi/doc-ru-config-values.yaml

```
@@ -169,6 +169,14 @@ properties:
       Структура, аналогичная `spec.nodeSelector` пода Kubernetes.
 
       Если ничего не указано или указано `false`, будет [использоваться автоматика](https://deckhouse.ru/products/kubernetes-platform/documentation/v1/#выделение-узлов-под-определенный-вид-нагрузки).
+  longtermPodAntiAffinity:
+    type: string
+    description: |
+      Устанавливает podAntiAffinity конфигурацию для longterm Prometheus по отношению к main Prometheus.
+      Доступные варианты: `Preferred` или `Required`.
+      `Preferred` правило позволит размещать longterm на одном узле с main, в случае необходимости.
+      `Required` правило предотвратит размещение longterm на одном узле с main, при любых условиях.
+      Если параметр не указан или установлен в `false`, по умолчанию используется вариант `Preferred`.
```

```suggestion
      Если параметр не указан или установлен в `false`, по умолчанию используется вариант `Preferred`.
```
Он не может быть установлен в false же - в спеке enum?
Значение параметра по умолнчанию будет присутствовать на странице автоматически из спеки.

---

Не может, из изначальной релизации затесалось.